internal/operators/pm_from_file_test.go

```diff
@@ -0,0 +1,27 @@
+package operators
+import (
+    "testing"
+
+    "github.com/corazawaf/coraza/v3/experimental/plugins/plugintypes"
+)
+func TestPmFromFileAlias(t *testing.T) {
```

Comment:
Does this describe the actual test?

I think the test only compares the `pm` and `pmfromfile` operators. Not the alias.

Reply:
@satwiksps This test can easily be extended testing the alias alongside `@pmfromfile` https://github.com/corazawaf/coraza/blob/9293be843709277bf32ca6e2cada40775a19ab4a/testing/engine/operators_with_files.go